clayui.com/src/components/Editor/index.js

```diff
@@ -78,12 +82,62 @@ const Editor = ({
 						title={collapseCode ? 'Expand' : 'Collapse'}
 					/>
 
-					{!collapseCode && (
+					{jspCode ? (
 						<>
-							{imports && <LiveEditor disabled value={imports} />}
+							<ClayTabs modern>
+								<ClayTabs.Item
+									active={activeTabKeyValue == 0}
```

Comment:
```suggestion
									active={activeTabKeyValue === 0}
```

lets make sure to always use triple equals `===`, otherwise this can lead to hard to diagnose bugs.

Reply:
Thought so, the [Tabs docs](https://clayui.com/docs/components/tabs.html) has double equals, I'll change that too then